portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -86,6 +88,37 @@ definition {
 		}
 	}
 
+	@description = "LPS-143552. Verify that remote app of type Custom Element can be displayed by portal instance"
+	@priority = "5"
+	test CustomElementCanBeDisplayedByInstance {
+		property portal.acceptance = "true";
+		property test.assert.warning.exceptions = "true";
+		property test.name.skip.portal.instance = "RemoteApps#CustomElementCanBeDisplayedByInstance";
+
+		task ("Create virtual instance") {
+			SAML.addVirtualInstances(virtualHosts = "www.able.com");
+		}
+
+		task ("Navigate to virtual instance") {
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@www.able.com");
+		}
+
+		task ("Assert custom remote app is present") {
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.viewTableEntries(
+				entryName = "Vanilla Counter",
+				entryType = "Custom Element");
+		}
+
+		task ("Navigate back to default instance") {
+			User.firstLoginPG();
```

Comment:
Use Navigator.openURL(); instead, skip the sign in process and just navigate to the baseURL.